v03_pipeline/lib/reference_data/clinvar.py

```diff
@@ -120,14 +125,25 @@ def download_and_import_latest_clinvar_vcf(
             reference_genome=reference_genome.value,
             drop_samples=True,
             skip_invalid_loci=True,
-            contig_recoding=reference_genome.contig_recoding(include_mt=True),
+            contig_recoding=reference_genome.contig_recoding(include_mt),
```

Comment:
🤔 I think there's a subtly here that makes this maybe not work the way we think.  The contig recoding is saying when I see `MT`, turn that into `chrM`.  So for `GRCh38`, if we don't do that, `MT` will stay `MT` and we'll drop those rows when we [filter for the standard contigs](https://github.com/broadinstitute/seqr-loading-pipelines/blob/a1564d7d4c1ac03a28129e60bd84e9603b3ced5c/v03_pipeline/lib/reference_data/dataset_table_operations.py#L59).  However, for `GRCh37`, the `MT` is already an `MT` and we don't use the recode, so we'll keep that row because `MT` is actually a standard contig.  

Looks like we can just do `~(ht.locus.contig == 'MT')` or `hl.filter_intervals(ht, [hl.parse_locus_interval('MT')]` here.  

Let me know if that makes sense.

Reply:
Ah I just read in the table, filtered for 'MT' and yep I see them. I'll use `~(ht.locus.contig == 'MT')` instead.